packages/cli/lib/services/dryrun.service.ts

```diff
@@ -30,7 +30,7 @@ class DryRunService {
 
         this.returnOutput = returnOutput;
     }
-    public async run(options: RunArgs, optionalEnvironment?: string, debug = false): Promise<string | void> {
+    public async run(options: RunArgs, optionalEnvironment?: string, optionalProviderConfigKey?: string, debug = false): Promise<string | void> {
```

Comment:
why are `optionalEnvironment` and `optionalProviderConfigKey` not part of `RunArgs`?

Reply:
Good question. When I was implementing I thought RunArgs was global to the CLI but it is just local to dry run service. In any case, updated with https://github.com/NangoHQ/nango/pull/2125/commits/4ab52803213d736e0b0e75067e55ff99511d16c9